qdrant-landing/content/documentation/concepts/filtering.md

```diff
@@ -2977,3 +2977,145 @@ Filtered points would be:
   { "id": 5, "city": "Moscow", "color": "green" }
 ]
 ```
+
+
+### Has vector
+
+*Available as of v1.13.0*
+
+This condition enables filtering by the presence of a given named vector on a point.
+
+For example, if we have two named vector in our collection.
+
+```http
+PUT /collections/{collection_name}
+{
+    "vectors": {
+        "image": {
+            "size": 4,
+            "distance": "Dot"
+        },
+        "text": {
+            "size": 8,
+            "distance": "Cosine"
+        }
+    },
+    "sparse_vectors": {
+        "sparse-image": {},
+        "sparse-text": {},
+    },
+}
+```
+
+Some points in the collection might have all vectors, some might have only a subset of them.
+
+<aside role="status">If your collection does not have named vectors, use an empty (<code>""</code>) name.</aside>
+
+This is how you can search for points which have the dense `image` vector defined:
+
+```http
+POST /collections/{collection_name}/points/scroll
+{
+    "filter": {
+        "must": [
+            { "has_vector": "image" }
+        ]
+    }
+}
+```
+
+```python
+from qdrant_client import QdrantClient, models
+
+client = QdrantClient(url="http://localhost:6333")
+
+client.scroll(
+    collection_name="{collection_name}",
+    scroll_filter=models.Filter(
+        must=[
+            models.HasVectorCondition(has_vector="image"),
+        ],
+    ),
+)
+```
+
+
+```typescript
+client.scroll("{collection_name}", {
```

Comment:
I checked this snippet at https://github.com/qdrant/qdrant-js/pull/91, and it's correct. Thanks, @agourlay!